src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationRunner.kt

```diff
@@ -81,166 +77,181 @@ class IntegrationRunner {
                 }
             }
 
-            integrations.forEach { (datasourceName, destinationsForDatasource) ->
+            integrationsMap.forEach { (datasourceName, destinationsForDatasource) ->
                 val datasource = datasources.getValue(datasourceName)
 
                 Multimaps.asMap(destinationsForDatasource).forEach { (destinationName, integrations) ->
-                    integrations.forEach { integration ->
-                        val destination = destinations.getValue(destinationName)
-
-                        logger.info("Running integration: {}", integration)
-
-                        val ds = getSourceDataset(datasource, integration)
-                        logger.info("Read from source: {}", datasource)
-                        //Only CSV and JDBC are tested.
-                        when (destination.writeDriver) {
-                            CSV_DRIVER -> ds.write().option("header", true).csv(destination.writeUrl)
-                            "parquet" -> ds.write().parquet(destination.writeUrl)
-                            "orc" -> ds.write().orc(destination.writeUrl)
-                            else -> toDatabase(integrationConfiguration.name, ds, destination, integration)
-                        }
-                        logger.info("Wrote to name: {}", destination)
+                    )
+                    val extIntegrations = integrations.filter { !it.gluttony } + integrations
+                            .filter { it.gluttony }
+                            .flatMap { integration ->
+
+                                val destination = destinations.getValue(destinationName)
+                                BasePostgresIterable(
+                                        StatementHolderSupplier(destination.hikariDatasource, integration.source)
+                                ) { rs ->
+                                    Integration(
+                                            rs.getString("description"),
+                                            rs.getString("query"),
+                                            rs.getString( "destination"))
```

Comment:
Where do these strings come from?

Reply:
I have a branch that centralizes all of these but we don't want to merge it quite yet since we're doing all these PDs this week:
https://github.com/openlattice/launchpad/pull/10